doc/v2/documentation_data_format.rst

```diff
@@ -242,27 +354,25 @@ replicates and plot error bars.
 Detailed field description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- ``observableId`` [STRING, NOT NULL, REFERENCES(observables.observableID)]
+- ``observableId`` [STRING, REQUIRED, REFERENCES(observables.observableID)]
 
   Observable ID as defined in the observable table described below.
 
-- ``preequilibrationConditionId`` [STRING OR NULL, REFERENCES(conditionsTable.conditionID), OPTIONAL]
-
-  The ``conditionId`` to be used for preequilibration. E.g. for drug
-  treatments, the model would be preequilibrated with the no-drug condition.
-  Empty for no preequilibration.
+- ``experimentId`` [STRING, REQUIRED, REFERENCES(experimentsTable.experimentID)]
 
-- ``simulationConditionId`` [STRING, NOT NULL, REFERENCES(conditionsTable.conditionID)]
+  Experiment ID as defined in the experiments table described below. This column may
+  have ``NA`` values, which are interpreted as *use the model as is*.
+  This avoids the need for "dummy" conditions and experiments if only a single
+  condition is required.
 
-  ``conditionId`` as provided in the condition table, specifying the condition-specific parameters used for simulation.
-
-- ``measurement`` [NUMERIC, NOT NULL]
+- ``measurement`` [NUMERIC, REQUIRED]
 
   The measured value in the same units/scale as the model output.
 
-- ``time`` [NUMERIC OR STRING, NOT NULL]
+- ``time`` [NUMERIC OR STRING, REQUIRED]
 
-  Time point of the measurement in the time unit specified in the SBML model, numeric value or ``inf`` (lower-case) for steady-state measurements.
+  Time point of the measurement in the time unit specified in the SBML model,
```

Comment:
Would consider it to be less confusing if ``inf`` was considered a numeric value here.

Reply:
"Finite numeric value or `inf`"? Since `-inf` would not be allowed?